translator/translate/otel/receiver/jmx/translator.go

```diff
@@ -134,14 +135,23 @@ func (t *translator) Translate(conf *confmap.Conf) (component.Config, error) {
 		cfg.JARPath = os.Getenv(jmxEnvJarPath)
 	}
 
-
 	if endpoint, ok := jmxKeyMap[common.Endpoint].(string); ok {
 		cfg.Endpoint = endpoint
 	}
 
 	cfg.TargetSystem = defaultTargetSystem
-	if targetSystem, ok := jmxKeyMap[targetSystem].(string); ok {
-		cfg.TargetSystem = targetSystem
+	targetSystems := ""
+	for _, jmxTarget := range jmxTargets {
```

Comment:
So, is the goal for this to filter out invalid target systems? Shouldn't that be handled by the schema?

Reply:
The idea is we need to find which target systems to look for unless we just search for all target systems. This is receiver code.